### PR TITLE
238-new-make-shuffle-mode.md: update against 4.4 release

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

File: posts/238-new-make-shuffle-mode.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ date: February 23, 2022
 I implemented new `--shuffle` option for `GNU make` to simulate
 non-deterministic build order in parallel makefiles.
 
+**UPDATE**: `--shuffle` mode was released as `GNU make 4.4` and can be
```
Comment (hartwork) [inline posts/238-new-make-shuffle-mode.md]:
```diff
--- as `GNU make 4.4`
+++ as part of GNU make 4.4
```
? The lone "as" seems a bit weird currently.
Also the use of backticks seems unexpected here because it's a name and version, not code or a command's name. Multiple cases like this below in both files.

File: posts/238-new-make-shuffle-mode.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ date: February 23, 2022
...
+used as:
+
+- `make --shuffle ...` from the command line or via, for interactive
```
Comment (hartwork) [inline posts/238-new-make-shuffle-mode.md]: Is there something missing after "via"?

File: posts/238-new-make-shuffle-mode.md [hunk 3]
```diff
@@ -8,6 +8,20 @@ date: February 23, 2022
...
+  use.
+- `GNUMAKEFLAGS=--shuffle` via environment variables for wider scale
+  builds.
+
+[Options Summary](https://www.gnu.org/software/make/manual/html_node/Options-Summary.html)
+has more details.
+
+It already found bugs in [30+ packages](/posts/249-an-update-on-make-shuffle.html)
+like `gcc`, `vim`, `ghc`, `subversion`, strace`, ispell` and others.
```
Comment (hartwork) [inline posts/238-new-make-shuffle-mode.md]: "strace" and "ispell" have unbalanced backticks:
```diff
--- strace`, ispell`
+++ `strace`, `ispell`
```

**Review by hartwork — COMMENTED** [review]
Found a few smaller bugs, please see below:

File: posts/238-new-make-shuffle-mode.md (hunk 1 above)
Comment (trofi) [reply posts/238-new-make-shuffle-mode.md]: Ah, good point! Updated.

File: posts/238-new-make-shuffle-mode.md (hunk 2 above)
Comment (trofi) [reply posts/238-new-make-shuffle-mode.md]: I don't remember. Dropped `via`.

File: posts/238-new-make-shuffle-mode.md (hunk 3 above)
Comment (trofi) [reply posts/238-new-make-shuffle-mode.md]: Good catch! Updated as well.

File: posts/238-new-make-shuffle-mode.md [hunk 4]
```diff
@@ -5,9 +5,22 @@ date: February 23, 2022
...
 # TL;DR:
 
-I implemented new `--shuffle` option for `GNU make` to simulate
+I implemented new `--shuffle` option for GNU make to simulate
 non-deterministic build order in parallel makefiles.
 
+**UPDATE**: `--shuffle` mode was released as part of GNU make 4.4 and
+can be used as:
+
+- `make --shuffle ...` from the command line for interactive use.
```
Comment (hartwork) [inline posts/238-new-make-shuffle-mode.md]: Maybe the last thing, I find "interactive use" a bit misleading here, interactive in my mind is when e.g. the software asks for confirmation or is using curses. Just my two cents.

Comment (trofi) [reply posts/238-new-make-shuffle-mode.md]: Yeah, I agree. Changed to just "`make --shuffle ...` from the command line.". How about that?

Comment (hartwork) [reply posts/238-new-make-shuffle-mode.md]: @trofi seems okay :+1:

**Review by hartwork — APPROVED** [review]

**Comment by trofi** [comment]
Pushed generated version as https://github.com/trofi/trofi.github.io/commit/e59b64c0136d246cab470e934c80744365d24da3.

Deployment says it's finished as well: https://github.com/trofi/trofi.github.io/actions/runs/3932389697

**Comment by hartwork** [comment]
@trofi seems so, yes, https://trofi.github.io/posts/238-new-make-shuffle-mode.html

**Comment by hartwork** [comment]
@trofi btw trofi.github.io seems to lack a redirect from http to https

**Comment by trofi** [comment]
Clicked through https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https. I hope I did not break it completely :)

**Comment by hartwork** [comment]
@trofi I think it works perfectly, now :+1:
